eGradebook.trace.db

```diff
@@ -685,3 +685,23 @@ Wrong user name or password [28000-200]
 	at org.h2.message.DbException.getJdbcSQLException(DbException.java:461)
 	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
 	... 46 more
+2021-02-25 22:16:49 jdbc[3]: exception
+org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabela "STUDENT" nie istnieje
+Table "STUDENT" not found; SQL statement:
+select * from student [42102-200]
+2021-02-25 22:22:52 jdbc[3]: exception
+org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabela "STUDENT" nie istnieje
+Table "STUDENT" not found; SQL statement:
+select * from student [42102-200]
+2021-02-25 22:28:43 jdbc[3]: exception
+org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabela "STUDENT" nie istnieje
+Table "STUDENT" not found; SQL statement:
+select * from student [42102-200]
+2021-02-25 22:28:50 jdbc[3]: exception
+org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabela "TEST" nie istnieje
+Table "TEST" not found; SQL statement:
+select * from test [42102-200]
+2021-02-25 22:28:58 jdbc[3]: exception
+org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabela "PERSON" nie istnieje
+Table "PERSON" not found; SQL statement:
+select * from person [42102-200]
```

Comment:
a co to za pliczek?
proszę zapoznać się z .gitignore ;)

Reply:
obraz krwawicy, potu i łez :p